src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -203,16 +202,26 @@ public function loadAnnotations(\ReflectionMethod|\ReflectionClass|\ReflectionPr
         }
 
         if ($reflector instanceof \ReflectionClass) {
-            yield from $this->getClassAnnotations($reflector);
+            yield from $this->getClassAttributes($reflector);
         }
         if ($reflector instanceof \ReflectionMethod) {
-            yield from $this->getMethodAnnotations($reflector);
+            yield from $this->getMethodAttributes($reflector);
         }
         if ($reflector instanceof \ReflectionProperty) {
-            yield from $this->getPropertyAnnotations($reflector);
+            yield from $this->getPropertyAttributes($reflector);
         }
     }
 
+    /**
+     * @deprecated since Symfony 6.4, use "loadAttributes()" instead
+     */
+    public function loadAnnotations(\ReflectionMethod|\ReflectionClass|\ReflectionProperty $reflector): iterable
+    {
+        trigger_deprecation('symfony/serializer', '6.4', 'Method "%s()" is deprecated, use "loadAttributes()" instead.', __METHOD__);
```

Comment:
Isn't it triggering the deprecation twice? One with the phpdoc and one with the `trigger_deprecation`

Reply:
I don't think so :thinking: If I remember correctly, the PHPDoc one is more for the IDE and DX than anything else